recipes/metis/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "metis" %}
+{% set version = "5.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: metis-5.1.0.tar.gz
+  md5: 5465e67079419a69e0116de24fce58fe
+  url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz
+
+build:
+  skip: true  # [win]
+  number: 0
+  binary_relocation: true
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+
+test:
+  commands:
+    - graphchk
+    - cmpfillin -h
+    - mpmetis -h
+    - gpmetis -h
+    - ndmetis -h
+    - gpmetis -h
+    - m2gmetis -h
+
+about:
+  home: http://glaros.dtc.umn.edu/gkhome/metis/metis/overview
+  license: GPLv3
```

Comment:
@tkelman thank you for noting. I've changed it in the [feedstock](https://github.com/conda-forge/metis-feedstock/commit/5298f1adf97bea72d2e04acec7f9d884665811f7)